usaspending_api/common/mixins.py

```diff
@@ -254,6 +256,70 @@ class SuperLoggingMixin(LoggingMixin):
 
     events_logger = logging.getLogger("events")
 
+    """Mixin to log requests - customized to disable DB logging, remove this method to re-enable"""
+    def initial(self, request, *args, **kwargs):
+        """Set current time on request"""
+
+        # check if request method is being logged
+        if self.logging_methods != '__all__' and request.method not in self.logging_methods:
+            super(LoggingMixin, self).initial(request, *args, **kwargs)
+            return None
+
+        # get IP
```

Comment:
Arman and I chatted about this, and he made a good case for how an IP address fits into our overall understanding of how the API is being used. We arrived at leaving this but adding a story that calls for disposing of the logs at regular intervals.